karma.conf.js

```diff
@@ -98,14 +99,17 @@ module.exports = function(config) {
         // Cover source files when not debugging tests. Otherwise, omit coverage instrumenting to get
         // uncluttered source maps.
         rules: webpackConfig.module.rules.concat([config.singleRun ? {
-          test: /\.js$/,
+          enforce: 'post',
+          test: /\.(js|ts)?$/,
```

Comment:
Why is there a `?` here? We should only be running on js or ts files, and nothing should end with just `.`

Reply:
you're right...I had an `x?` there for jsx/tsx